### DISCUSSION TITLE
Change Kubernetes Helm Chart

### DISCUSSION THREAD

**Discussion by MrDuartePT** [goauthentik/authentik#3620] [General]
I tried to deploy authentik in kubernetes on raspberry pi and find out the bitami container for redis and postgresql doesn't support arm. It's would be possible to change to container would support arm.

**Comment by BeryJu** [comment]
https://github.com/goauthentik/helm/issues/16

**Reply by MrDuartePT** [reply]
Thanks for show me that issue I don't have found it. Because it a year old I will try with the latest version in the tag to see if works.

**Reply by MrDuartePT** [reply]
In not possible to use the latest release authentik-server never starts.
